roles/confluent.variables/defaults/main.yml

```diff
@@ -112,6 +112,9 @@ ssl_keystore_key_password: ""
 ### Keystore Password for host specific keystore. Used with ssl_provided_keystore_and_truststore: true. May set per host if keystores have unique passwords
 ssl_keystore_store_password: ""
 
+### Keystore source alias for host specific certificate if keystore contains more than one certificate. Used with ssl_provided_keystore_and_truststore: true. May set per host, or use inventory_hostname variable eg "{{inventory_hostname}}"
```

Comment:
```suggestion
### Keystore source alias for host specific certificate. Only required if keystore contains more than one certificate. Used with ssl_provided_keystore_and_truststore: true. May set per host, or use inventory_hostname variable eg "{{inventory_hostname}}"
```

I want to make this more clear it is not a required variable

Reply:
agreed!